tests/unit/responses/elsevier/j.nima.2019.162728_expected.yml

```diff
@@ -0,0 +1,261 @@
+      langtype="en"><authors><author><surname>Golubeva</surname><given-name>M.</given-name></author><et-al/></authors></contribution><host><issue><series><title><maintitle>J.
+      Phys.: Conf. Ser.</maintitle></title><volume-nr>798</volume-nr></series><date>2017</date></issue><article-number>012073</article-number></host></reference><source-text
+      id="afs5">M. Golubeva et al., Journal of Physics: Conf. Series798 (2017) 012073</source-text></bib-reference>'
+    source: Elsevier B.V.
+  reference:
+    publication_info:
+      journal_title: 'J. Phys.: Conf. Ser.'
+      journal_volume: '798'
+      year: 2017
+    label: '5'
+    authors:
+    - full_name: Golubeva, M.
+      inspire_role: author
+- raw_refs:
+  - schema: Elsevier
+    value: '<bib-reference id="b6"><label>[6]</label><other-ref id="sb6"><textref>N.
+      Karpushkin, et al. Accepted at NIM A: <inter-ref id="interref1" href="https://doi.org/10.1016/j.nima.2018.10.054"
+      type="simple">https://doi.org/10.1016/j.nima.2018.10.054</inter-ref>.</textref></other-ref></bib-reference>'
+    source: Elsevier B.V.
+  reference:
+    dois:
+    - 10.1016/j.nima.2018.10.054
+    label: '6'
+    misc:
+    - 'N. Karpushkin, et al. Accepted at NIM A:'
+- raw_refs:
+  - schema: Elsevier
+    value: <bib-reference id="b7"><label>[7]</label><other-ref id="sb7"><textref><inter-ref
+      id="interref2" href="http://afi.jinr.ru" type="simple">http://afi.jinr.ru</inter-ref>.</textref></other-ref></bib-reference>
+    source: Elsevier B.V.
+  reference:
+    persistent_identifiers:
+    - schema: HDL
+      value: http://afi.jinr.ru
+    label: '7'
+- raw_refs:
+  - schema: Elsevier
+    value: <bib-reference id="b8"><label>[8]</label><reference id="sb8"><contribution
+      langtype="en"><authors><author><surname>Mikhaylov</surname><given-name>V.</given-name></author><et-al/></authors></contribution><host><issue><series><title><maintitle>Nucl.
+      Instrum. Meth.</maintitle></title><volume-nr>A912</volume-nr></series><date>2018</date></issue><pages><first-page>241</first-page><last-page>244</last-page></pages></host></reference><source-text
+      id="afs8">V. Mikhaylov et al., Nucl.Instrum.Meth. A912 (2018) 241-244</source-text></bib-reference>
+    source: Elsevier B.V.
+  reference:
+    publication_info:
+      journal_title: Nucl. Instrum. Meth.
+      journal_volume: A912
+      year: 2018
+      page_start: '241'
+    label: '8'
+    authors:
+    - full_name: Mikhaylov, V.
+      inspire_role: author
+- raw_refs:
+  - schema: Elsevier
+    value: <bib-reference id="b9"><label>[9]</label><other-ref id="sb9"><textref><inter-ref
+      id="interref3" href="http://dpnc.unige.ch/bravar/exp/NA61/Bravar-DRS.pdf" type="simple">http://dpnc.unige.ch/bravar/exp/NA61/Bravar-DRS.pdf</inter-ref>.</textref></other-ref></bib-reference>
+    source: Elsevier B.V.
+  reference:
+    persistent_identifiers:
+    - schema: HDL
+      value: http://dpnc.unige.ch/bravar/exp/NA61/Bravar-DRS.pdf
```

Comment:
this is wrong, it's not a handle (`HDL`). I think you need to exclude stuff starting with `http://` (for unrecognized domains) and add them as URLs instead (to be done in the builder).